linked_list/linked_list.py

```diff
@@ -0,0 +1,66 @@
+class Node:
```

Comment:
Move it to some different file called node.py. So we can use it other places as Stack.